docs/en/operation/Configuration.md

```diff
@@ -142,32 +143,44 @@ master's `alluxio-site.properties` file.
 After restarting the cluster, all jobs will automatically set `alluxio.user.file.writetype.default`
 to `CACHE_THROUGH`.
 
-Clients can ignore or overwrite the cluster-wide default values by following the approaches described in
-[Configure Alluxio for Applications](#configure-applications) to overwrite the same properties.
+Clients can ignore or overwrite the cluster-wide default values by following the approaches
+described in [Configure Applications](#configure-applications) to overwrite the same properties.
 
 > Note that, before version 1.8, `${ALLUXIO_HOME}/conf/alluxio-site.properties` file is only loaded
 by Alluxio server processes and will be ignored by applications interacting with Alluxio service
 through Alluxio client, unless `${ALLUXIO_HOME}/conf` is on applications' classpath.
 
 ### Path Defaults
 
-Since version 2.0, Alluxio administrators can set default client side configurations for Alluxio paths.
-FileSystem client operations have options, FileSystem options are derived from client side configuration
-properties. Only these configuration properties can be set as as path defaults.
+Since version 2.0, Alluxio administrators can set default client side configurations for Alluxio
+paths.
+Filesystem client operations have options, Filesystem options are derived from client side
```

Comment:
grammar? Make it two sentences, use a semi-colon, merge?

Reply:
updated